src/services/addressResolver/index.test.ts

```diff
@@ -0,0 +1,20 @@
+import { getIdentityName } from "./index";
+
+it("should get the expected names from resolver", async () => {
```

Comment:
Additional test:
it("should return the first name if it can be found with any resolver")
it("should return undefined if it cannot be resolved anywhere")
it("should return undefined with empty resolver list")